test/integration/regression-tests.postman_collection.json

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
Can we please be more specific with the file name saying that these are v2 regression tests? Future regression tests will all be in their own collections.